build.gradle.kts

```diff
@@ -39,6 +39,12 @@ dependencies {
 
     val kotlinVersion = rootProject.extra["kotlinVersion"]
 
+    implementation("org.jetbrains.kotlinx:dataframe:0.13.1")
+    implementation("org.jetbrains.lets-plot:lets-plot-kotlin:4.7.0")
+
+    implementation("org.jetbrains.lets-plot:lets-plot-kotlin-jvm:4.7.0")
+    implementation("org.jetbrains.lets-plot:lets-plot-image-export:4.3.0")
+    implementation("org.jetbrains.lets-plot:lets-plot-batik:4.3.0")
     implementation("org.biokotlin:biokotlin:0.14")
     implementation("com.github.ajalt.clikt:clikt:4.2.0")
 
```

Comment:
See below implementation("org.jetbrains.kotlin:kotlin-test:1.9.10")
Maybe that should use ${kotlinVersion}
I know I did it, but not sure why

Reply:
I changed to use kotlinVersion